kolibri/plugins/user/assets/src/views/SignInPage/index.vue

```diff
@@ -281,10 +281,15 @@
         return urls['kolibri:core:guest']();
       },
       backgroundImageStyle() {
-        return {
-          backgroundColor: this.$themeTokens.primary,
-          backgroundImage: `linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)), url(${require('./background.jpg')})`,
-        };
+        if (global.kolibriTheme.signInBackground) {
```

Comment:
Would prefer to consolidate all references to the global API in the theme module, then reference that through imports where it ends up being used - just to decouple our backend injection from frontend usage.